### PR TITLE
Feature/nr 111940 nrai demo mode

### PR DESCRIPTION
**Pull request by dsellarsnr**

### CODE REVIEW

File: shared/ui/Stream/CodeErrorNav.tsx [hunk 1]
```diff
@@ -602,18 +606,18 @@ export function CodeErrorNav(props: Props) {
 					await dispatch(
 						updateCodeError({
 							...derivedState.codeError,
-							accountId: errorGroupResult.accountId,
-							title: errorGroupResult.errorGroup?.title || "",
-							text: errorGroupResult.errorGroup?.message || undefined,
+							// accountId: errorGroupResult.accountId,
```
Comment (bcanzanella) [inline shared/ui/Stream/CodeErrorNav.tsx:609]: these commented out are ok? if so, perhaps just remove them all?

File: shared/ui/src/components/Pane.tsx [hunk 2]
```diff
@@ -233,14 +246,44 @@ export const PaneHeader = React.memo((props: PropsWithChildren<PaneHeaderProps>)
...
+			dispatch(setDemoMode(nextDemoMode));
+			captured = "";
+		}
+	};
+
 	if (props.noDropdown) {
 		return (
 			<PaneHeaderRoot
+				onClick={demoClick}
+				onContextMenu={demoClick}
```
Comment (bcanzanella) [inline shared/ui/src/components/Pane.tsx:278]: does this need to be guarded at all? aka will non-demo folks be able to trigger this?

File: shared/ui/store/codeErrors/api/CodeErrorsApi.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import {
```
Comment (bcanzanella) [inline shared/ui/store/codeErrors/api/CodeErrorsApi.ts:1]: nice!

**Review by bcanzanella — CHANGES_REQUESTED** [review]
just a couple comments/questions

File: shared/ui/src/components/Pane.tsx (hunk 2 above)
Comment (dsellarsnr) [reply shared/ui/src/components/Pane.tsx:278]: i made it a cheat code, you have to mouse click left, right, left, right, left, left

File: shared/ui/Stream/CodeErrorNav.tsx (hunk 1 above)
Comment (dsellarsnr) [reply shared/ui/Stream/CodeErrorNav.tsx:609]: the server call is pretty close to matching the type - only allows updates to title, text, and stackTraces fields so `objectInfo` update never did anything here.

**Review by bcanzanella — APPROVED** [review]

**Pull request merged**